BenchmarkSample/ui/src/main/java/com/example/benchmark/ui/MainActivity.kt

```diff
@@ -24,35 +23,10 @@ class MainActivity : AppCompatActivity() {
     }
 }
 
-class Holder(root: View) : RecyclerView.ViewHolder(root) {
-    private val textView: TextView = root.findViewById(R.id.label)
-
-    fun bind(string: String) {
-        textView.text = string
-    }
-
-    companion object {
-        fun create(parent: ViewGroup): Holder {
-            val inflater = LayoutInflater.from(parent.context)
-            return Holder(inflater.inflate(R.layout.item_view, parent, false))
-        }
-    }
-}
-
-class SampleAdapter : ListAdapter<String, Holder>(StringDiffCallback) {
-    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): Holder {
-        return Holder.create(parent)
-    }
-
-    override fun onBindViewHolder(holder: Holder, position: Int) {
-        holder.bind(getItem(position))
-    }
-}
-
-private object StringDiffCallback : DiffUtil.ItemCallback<String>() {
-    override fun areItemsTheSame(oldItem: String, newItem: String): Boolean =
-        oldItem == newItem
+// stable random for having the same results
+private val random = Random(0)
```

Comment:
this randomness isn't stable for two reasons:

1) benchmark will run an arbitrary number of loops. Because this work is inconsistent, the number of loops will affect measured number

2) random isn't reset between benchmarks, so it's easy to have state leaked between them.


randomness in benchmarks should either be:

A) Static, stably seeded data created before the benchmark (each iteration sees same exact input)
B) Random seeded per iteration (functionally same - resulting in the same data each iteration)